dev/com.ibm.ws.jdbc_fat_driver/test-applications/jdbcapp/src/jdbc/fat/driver/web/JDBCDriverManagerServlet.java

```diff
@@ -401,4 +407,33 @@ public void testGetSetLoginTimeout() throws Exception {
         assertEquals("Login timeout should be 0 regardless of what is done to DriverManager", 0, fatDriverDS.getLoginTimeout());
     }
 
+    //Test that you are unable to set the logwriter on the DataSource and that the getLogWriter method returns null
+    @Test
+    public void testGetSetLogWriter() throws Exception {
+        try {
+            fatDriverDS.setLogWriter(new PrintWriter(System.out));
+        } catch (SQLFeatureNotSupportedException ex) {
+        } //expected
+        assertNull("The getLogWriter method should always return null when using Driver", fatDriverDS.getLogWriter());
+    }
+
+    //Test that ensures unwrapping a DataSource backed by Driver to the underlying Driver interface is not possible
+    @Test
+    @ExpectedFFDC({ "java.sql.SQLException" })
+    public void testUnwrapDriver() throws Exception {
+        assertFalse("fatDriverDS should not wrap Driver", fatDriverDS.isWrapperFor(Driver.class));
+        try {
+            fatDriverDS.unwrap(Driver.class);
+            fail("Should not be able to unwrap to the Driver interface");
+        } catch (SQLException ex) {
+            if (!ex.getMessage().contains("DSRA9122E"))
+                throw ex;
+        }
+
+        //It should however still be able to be unwrapped to an interface impl'd by WsJdbcDataSource
+        assertTrue("fatDriverDS should wrap CommonDataSource", fatDriverDS.isWrapperFor(CommonDataSource.class));
+        CommonDataSource ds = fatDriverDS.unwrap(CommonDataSource.class);
+        assertEquals("The WSJdbcDataSource instance should have been returned by the call to unwrap", fatDriverDS, ds);
```

Comment:
Sorry if I misled you with the wording of my previous comment.  For == comparison in an assert, you need to use assertSame rather than assertEquals.  Again, this is a minor comment, and if you decide not to change it, that's okay